spin/cmds/pip.py

```diff
@@ -0,0 +1,34 @@
+import click
+
+from .util import run as _run
+
+
+@click.command()
+@click.option(
+    "-v", "--verbose", is_flag=True, help="Print detailed build and installation output"
+)
+@click.argument("-e", "--editable", is_flag=True, help="Install in editable mode")
+@click.argument("pip_args", nargs=-1)
+def install(pip_args, verbose=False, editable=True):
+    """💽 Install package and also build.
+
+    pip arguments are passed through in the same manner as you would pip e.g.:
+
+        spin install -- -r requirements.txt --editable false
+
+    which would be translated to the following pip command:
+
+        pip install -r requirements.txt
+
+    The package is by default installed in editable mode, but you can disable this
+    by passing the `--editable` flag.
+    """
+    pip_args = list(pip_args)
+    pip_cmd = ["pip", "install", "."]
+    if editable:
+        pip_args += ["--no-build-isolation", "--editable"]
```

Comment:
Maybe `--no-build-isolation` should be used in both cases? I.e.,`pip_cmd = ["pip", "install", ".", "--no-build-isolation"]`

Reply:
Sure fair.